_quarto.yml

```diff
@@ -31,7 +31,7 @@ website:
         href: "https://github.com/geojupyter"
 
   page-footer:
-    left: "© 2024 GeoJupyter Contributors"
+    left: "© 2024 GeoJupyter Contributors.<br/>_GeoJupyter is not affiliated with [Project Jupyter](https://jupyter.org)_."
```

Comment:
I would slightly soften this as it makes it sound like it's more remote than we actually are - I'd suggest something like

```
GeoJupyter is a community effort that includes many Jupyter contributors and follows all Jupyter guidelines, but is currently not an official Jupyter subproject.
```

How does that sound?

Reply:
Love it, thank you @fperez !

Reply:
062cbde